sdcm/sct_config.py

```diff
@@ -1229,6 +1229,10 @@ class SCTConfiguration(dict):
              env="SCT_MGMT_SNAPSHOTS_PREPARER_PARAMS", type=dict_or_str,
              help="Custom parameters of c-s write operation used in snapshots preparer"),
 
+        dict(name="one_one_restore_cluster_bootstrap_duration",
```

Comment:
Why would you need to pass this variable as a config? I do not think config should be used for transferring data between siren and SCT. This is just a value used for results reporting